src/lib/utils.ts

```diff
@@ -4,3 +4,7 @@ import { twMerge } from "tailwind-merge";
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+// this helper converts the numbers to USD currency format for further use in project :)
+export function FormatToUSD(input:any) {
```

Comment:
`FormatToUSD` should start with a small letter i.e. `formatToUSD`. Also, we should not hard code currency to USD. Please add another parameter in which we can define the currency and our locale will be mapped to that currency.

Eg: if we provide `INR` as currency then local will be `en-IN`

Please don't use `any` type. input can be of either `number` or `string`

Reply:
Also add proper jsdoc comment for documentation

Reply:
solved